GeneratorInterface/GenFilters/interface/LHEGenericFilter.h

```diff
@@ -49,8 +49,10 @@ class LHEGenericFilter : public edm::EDFilter {
   
   edm::EDGetTokenT<LHEEventProduct> src_;
   int numRequired_;                // number of particles required to pass filter
-  bool acceptMore_;                // if true (default), accept numRequired or more.
-                                   // if false, accept events with exactly equal to numRequired.
+  std::string acceptLogic_;        // LT  meaning <
```

Comment:
did you mean something like this
https://github.com/perrozzi/cmssw/commit/a4863f07bf8000836de45b683673ac998d6b8a4d